test/terra/backends/aer_simulator/test_truncate.py

```diff
@@ -135,3 +137,32 @@ def test_delay_measure(self):
         metadata = result.results[0].metadata
         self.assertIn('measure_sampling', metadata)
         self.assertTrue(metadata['measure_sampling'])
+
+    def test_parameterized_circuits(self):
+        """Test truncation for parameterized circuits"""
+        backend = self.backend()
+        circuit = QuantumCircuit(10, 10)
+        circuit = circuit.compose(RealAmplitudes(5), range(5))
+        circuit = transpile(circuit, backend)
+        for q in range(5):
+            circuit.measure(q, q)
+        
+        num_of_param_sets = 3
+        param_map = {}
+        for param in circuit.parameters:
+            param_values = [ np.random.random() for _ in range(num_of_param_sets) ]
+            param_map[param] = param_values
+
+        resolved_circuits = [circuit.bind_parameters({param: param_map[param][set_idx] for param in circuit.parameters})
+                             for set_idx in range(num_of_param_sets)]
+
+        result = backend.run(circuit, parameter_binds=[param_map]).result()
+        self.assertSuccess(result)
+
+        result_without_parameters = backend.run(resolved_circuits).result()
+        self.assertSuccess(result_without_parameters)
+
+        for actual_result, expected_result in zip(result.results, result_without_parameters.results):
+            metadata = actual_result.metadata
+            self.assertEqual(metadata["active_input_qubits"], [q for q in range(5)])
+            self.assertDictAlmostEqual(actual_result.data.counts, expected_result.data.counts, )
```

Comment:
Would it be possible/better to run a statevector simulator, to reduce the possible variation?  I'm slightly concerned that comparing two counts distributions is naturally going to be a bit flaky due to the randomness in the simulator (or is the simulator seeded before each call?), whereas comparing the statevectors is only subject to floating-point tolerance issues.

Reply:
I have moved this test to https://github.com/Qiskit/qiskit-aer/blob/47f2765b8c117e8f29792c45f50f64b80b32136f/test/terra/backends/test_parameterized_qobj.py, and then used statevectors for assertions.